how-to/security/apparmor.md

```diff
@@ -225,6 +211,43 @@ Profiles are meant to provide security and so can't be too permissive. But often
   * To mitigate the drawbacks of above approaches, **local includes** were introduced, adding the ability to write arbitrary rules that not run into issues during upgrades that modify the packaged rule.
   * The files can be found in `/etc/apparmor.d/local/` and exist for the packages that are known to sometimes need slight tweaks for special setups.
 
+
+## Disabling or Re-enabling AppArmor
+
+Starting with Ubuntu 24.04 and later, the AppArmor services are baked into the Ubuntu Kernel, and require special steps to fully disable or re-enable.
+
+### Disable AppArmor
+
+> WARNING! Disabling AppArmor reduces the security of your system! You should only disable apparmor if you understand the security implications 
+> of disabling the service!
+
+To disable AppArmor, you must do the following:
+
+* Stop and disable the `apparmor.service` in SystemD to prevent AppArmor profiles from being loaded:
```

Comment:
Due to AssertPathIsReadWrite=/sys/kernel/security/apparmor/.load
We actually could "only" document the module argument now.
That will avoid the service to be enabled/disables - less steps, less mistakes.

Also re-enabling it will be only one step again.
WDYT - should we just remove all about apparmor.service?

Reply:
Because disabling it with a kernel parameter works in the old style as well.
There you could provide the parameter via a conf file, but since disabling security is nothing you'd want to make too trivial just and only documenting the kernel parameter might be better readable and works everywhere.

Reply:
If we don't need `apparmor.service` to be disabled manually then we can remove that bit and just use the unloading of the module.  Same for re-enabling, because then the service works as intended if the module is present or not.

Reply:
If we don't need `apparmor.service` to be disabled manually then we can remove that bit and just use the unloading of the module.  Same for re-enabling, because then the service works as intended if the module is present or not.